scripts/ci/travis/build-release

```diff
@@ -24,3 +24,4 @@ else
   echo -e 'WARN: Should not be here => Branch ['$TRAVIS_BRANCH']  Tag ['$TRAVIS_TAG']  Pull Request ['$TRAVIS_PULL_REQUEST']'
   ./gradlew build --stacktrace
 fi
+
```

Comment:
extra line

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

tl;dr: POSIX, so why not?

Reply:
ah, my bad. My editor config confused me, we don't need 2 extra blank lines at the end.